index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```

Comment:
- [ ] **Required:** Please take some time to review your whole design before your next submission. 
It seems to be broken.

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/159551041-5f741d7f-2511-4b01-adc7-0dcfaa146f2d.png) | ![image](https://user-images.githubusercontent.com/27709832/159551383-a6146448-6db7-47ec-b9b9-2ee9b2a2c05f.png)

Reply:
- [ ] Please work on your design as required by the previous reviewer.
- [ ] Please make sure you match the design given in Figma.

[ Suggestions ]
- Kindly remove the background image you are using in the body tag in your CSS styles and place it in the headline section as that's the section that is supposed to have the background image
- Also set the `background-repeat: no-repeat;` style on the same headline section styles to stop the background image from repeating on the page